src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -64,6 +64,7 @@
 import com.google.javascript.jscomp.modules.ModuleMapCreator;
```

Comment:
please drop the now-no-op changes to this file

Reply:
done